MIDIchlorians/MIDIchlorians/AudioManager.swift

```diff
@@ -29,14 +29,20 @@ struct AudioManager {
 
     //call this to play audio with single directory
     //returns success
-    func play(audioDir: String) -> Bool {
+    func play(audioDir: String, bpm: Int? = nil ) -> Bool {
```

Comment:
I'm thinking we may need it for looping audio, can remove now since it's not being used